android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletDataFilesInstallerUtil.java

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.crypto_wallet.util;
+
+import org.chromium.base.annotations.CalledByNative;
+
+/**
+ * Class that is used by wallet_data_files_installer.cc to determine, do we need to download
```

Comment:
```suggestion
 * Class that is used by wallet_data_files_installer.cc to determine if we need to download
```

Reply:
thanks, the proposed text sounds better.

fixed at https://github.com/brave/brave-core/pull/19086/commits/54818d8dcc99e3f267ebe48dde6da4ba6625545f#diff-c7e4d99d837ddf5c13bf6fdc72cc6ae4b7b172941084c323bc6a424d0843a0b1R11